api/metrics/community_test.py

```diff
@@ -44,6 +44,34 @@ def test_population():
 
     assert metric.population(year=2013, segment="all") == [], "Should have no results for 2013."
 
+def test_income():
+    income_table = [
+        {
+            "area_number": 1,
+            "period_end_year": 2019,
+            "segment": "all",
+            "value": 13000
+        },
+        {
+            "area_number": 2,
+            "period_end_year": 2019,
+            "segment": "all",
+            "value": 27000
+        },
+        {
+            "area_number": 1,
+            "period_end_year": 2010,
+            "segment": "all",
+            "value": 10000
+        }
+    ]
+    con, cur = create_test_db(
+        scripts=[
+            "./pipeline/load/income.sql"
+        ],
+        tables={
+            "income": income_table
+        }
```

Comment:
What happened to the test assertions? When I looked yesterday I saw you had them in this method, maybe they got removed while fixing merge conflicts or something.

Reply:
Oh darn, will look into it

Reply:
As of right now, would you be okay with having identical metric tests in this file for the income and population tests? Made it identical since the methods are almost the same and that it seems like these tests are unit tests since they don't actual check into the database but an in-memory one.

Reply:
Yeah it's fine for them to be almost identical!

And yes, you're right that it's testing an in-memory database using the test data you declared. Just helps us test the metric logic on a controlled (and made up) dataset.